ansible/roles/ansible-tvault-contego-extension/tasks/install_rhel.yml

```diff
@@ -33,3 +58,9 @@
         - python3-tvault-contego
      state: latest
   when: PYTHON_VERSION=="python3"
+
+- name: Remove recently installed openstack-{{OPENSTACK_DIST}} repo
```

Comment:
Is this change intentional?

Reply:
Actually, it's fixed into the **dev-stable/4.2 branch**, this change is yet needed to merge into the **Master branch**